dashboard-api/setup_db.sh

```diff
@@ -0,0 +1,2 @@
+./cockroach sql --host=cockroachdb:26257 --insecure -e "CREATE DATABASE vlab"
```

Comment:
The commands on this script could easily be executed by docker-compose. Moving the content of this script to docker-compose will allow us to have one less file and one less file to mount to the container.